env/archivist_mainnet

```diff
@@ -7,15 +7,12 @@ LOG_FILE=${BASE_PATH}/aleph-node.log
 POOL_LIMIT=100
 UNIT_CREATION_DELAY=300
 PORT=30333
-RPC_PORT=9933
-WS_PORT=9944
+RPC_PORT=9944
 METRICS_PORT=9615
 RUST_LOG=info
 SYNC=Fast
 VALIDATOR=false
-WS_MAX_CONNECTIONS=10000
```

Comment:
Can you please remind me whether this was deliberate we added neither in https://github.com/Cardinal-Cryptography/aleph-node-runner/pull/53/files#diff-a5b71bd95feaf9cd2975e5c4036d869439338234db7e67523c64b46ed9c4f6cd nor in this PR, `RPC_MAX_CONNECTIONS=10000`? Technically that's a change in the behaviour, if this was intended then please ignore.

Reply:
I think I can restore it, makes sense to me. @kostekIV any thoughts?

Reply:
Yep, its better to have it